CRM/Core/PseudoConstant.php

```diff
@@ -211,15 +211,15 @@ public static function get($daoName, $fieldName, $params = [], $context = NULL)
     $dao = new $daoName();
     $fieldSpec = $dao->getFieldSpec($fieldName);
 
-    // Ensure we have the canonical name for this field
-    $fieldName = CRM_Utils_Array::value('name', $fieldSpec, $fieldName);
-
     // Return false if field doesn't exist.
     if (empty($fieldSpec)) {
       return FALSE;
     }
 
-    elseif (!empty($fieldSpec['pseudoconstant'])) {
+    // Ensure we have the canonical name for this field
+    $fieldName = CRM_Utils_Array::value('name', $fieldSpec, $fieldName);
```

Comment:
We should be replacing these where we can. It can't be done en-masse because the meaning isn't precisely the same so it requires judgment. In this case, I judge this to be just as good:

```suggestion
    $fieldName = $fieldSpec['name'] ?? $fieldName;
```

Reply:
Yep - I did try to avoid the rabbit hole of adding things unrelated to the thing I was trying to deprecate (passing non-arrays into that function) but I've just fixed this one for you.

I've given up on my deprecation now though - turns out to be pretty widespread :-(

Reply:
Thanks.

I'm just saying we need a new code pattern in our heads so that when we go to type "access an array value if it exists" we default to using `??` instead of `CRM_Utils_Array`.